src/robot/htmldata/rebot/log.js

```diff
@@ -54,6 +54,12 @@ function expandElement(item) {
     var children = element.children('.children');
     // .css is faster than .show and .show w/ callback is terribly slow
     children.css({'display': 'block'});
+    // in rare cases on large logs concurrent expanding fails => retry
+    if (children.css('display')!='block') { 
```

Comment:
Please add spaces around `!=` to be consistent with rest of the code.

Reply:
done